File: Inheritance.php
<?php
class ParentClass {
    public function greet() {
        echo "Hello from Parent\n";
    }
}

class ChildClass extends ParentClass {
    public function greet() {
        echo "Hello from Child\n";
    }
}

$child = new ChildClass();
$child->greet();
?>

File: Inheritance.cs
using System;

class ParentClass {
    public virtual void Greet() {
        Console.WriteLine("Hello from Parent");
    }
}

class ChildClass : ParentClass {
    public override void Greet() {
        Console.WriteLine("Hello from Child");
    }
}

class Program {
    static void Main() {
        ChildClass child = new ChildClass();
        child.Greet();
    }
}
